mlrun/runtimes/local.py

```diff
@@ -305,11 +305,23 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
                 tmp=tmp,
                 host=socket.gethostname(),
             )
-            fn = self._get_handler(handler, context)
-            global_context.set(context)
-            sout, serr = exec_from_params(fn, runobj, context)
-            log_std(self._db_conn, runobj, sout, serr, skip=self.is_child, show=False)
-            return context.to_dict()
+            try:
```

Comment:
there is already try/except in the calling function in base line 443, why isn't it good enough ?